ci/docker/runtime_functions.sh

```diff
@@ -353,7 +369,18 @@ sanity_check() {
 }
 
 
+
+before_unittest() {
```

Comment:
I don't think this is necessary since it can be retrieved from the log and adds boilerplate to the the functions

Reply:
I didn't see an easy way to see the instance id from the logs or locate where it's executing. What's the way you propose? The logs are huge already it's a good thing to have and doesn't create any problems. I would like to have it.

Reply:
I open the overall log and search for the [ut-cpp-gpu], for example. The first occurence contains the machine name.

I would rather have this as part of the build.py than in the runtime functions. They're supposed to only have logic and not boilerplate

Reply:
that is not very comfortable, we should make it easy to investigate problems.

Reply:
In how far is it less comfortable to have it in build.py?

Reply:
The machine name is the node name, this prints the instance id, so you can ssh directly in case you have a corefile for example.

Reply:
having in build.py is also fine for me